store/course/store.ts

```diff
@@ -1,81 +1,25 @@
-import type { RequestState } from '@/store/types';
-import { addObjectInfoToList } from '@/store/utils';
 import { useUnitStore } from '@/store/unit/store';
-import { unitDTOToUnit } from '@/store/unit/utils';
-import type { CourseDTO } from './types';
-import { courseDTOToCourse } from './utils';
+import { unitDTOToUnit } from '@/services/unit/utils';
+import { courseDTOToCourse } from '../../services/course/utils';
```

Comment:
Should we use `@` alisan here?

```suggestion
import { courseDTOToCourse } from '@/services/course/utils';
```